addons/controllers/clusterbootstrap_controller_test.go

```diff
@@ -55,22 +55,22 @@ var _ = Describe("TanzuClusterBootstrap Reconciler", func() {
 
 		BeforeEach(func() {
 			clusterName = "test-cluster-tcbt"
-			clusterResourceFilePath = "testdata/test-cluster-tanzu-cluster-bootstrap.yaml"
+			clusterResourceFilePath = "testdata/test-cluster-cluster-bootstrap.yaml"
```

Comment:
Nit: The name `test-cluster-cluster-bootstrap.yaml` looks a bit weird.

Reply:
+1